omod/src/main/resources/config.xml

```diff
@@ -176,6 +176,8 @@
 		WaiverType.hbm.xml
 		WaiverTypeRole.hbm.xml
 		EhrHospitalWaiver.hbm.xml
+        PatientRegimen.hbm.xml
+
 
```

Comment:
I haven't seen the hibernate config file for the Cycle model and it's equivalent config file entry 
```suggestion
Cycle.hbm.xml
```